docs/prod-requirements.md

```diff
@@ -1,33 +1,70 @@
 # Requirements for Zulip in production
 
-* Server running Ubuntu Trusty
-* At least 2 CPUs for production use with 100+ users
-* At least 4GB of RAM for production use with 100+ users.  We **strongly
-  recommend against installing with less than 2GB of RAM**, as you will
-  likely experience OOM issues.  In the future we expect Zulip's RAM
-  requirements to decrease to support smaller installations (see
-  [https://github.com/zulip/zulip/issues/32](https://github.com/zulip/zulip/issues/32)).
-* At least 10GB of free disk for production use (more may be required
-  if you intend to store uploaded files locally rather than in S3
-  and your team uses that feature extensively)
-* Outgoing HTTP(S) access to the public Internet.
-* SSL Certificate for the host you're putting this on
-  (e.g. zulip.example.com).  The installation instructions contain
-  documentation for how to get an SSL certificate for free using
-  [LetsEncrypt](https://letsencrypt.org/).
+Note that if you just want to play around with Zulip and see what it looks
+like, it is easier to install it in a development environment
+following [these
+instructions](readme-symlink.html#installing-the-zulip-development-environment),
+since then you don't need to worry about setting up SSL certificates and an
+authentication mechanism. Or, you can check out the
+[developers' chatroom](http://zulip.tabbott.net/) (a public, running Zulip
+instance).
+
+## Domain
+
+You should already have a full qualified domain name available for your Zulip
+production instance. In order to generate valid SSL certificates with Let's
+Encrypt, and to enable other services such as Google Authentication, you'll
+need to update the domains A record to point to your production server.
+
+## Operating System
+
+Only Ubuntu 14.04 Trusty is supported for running Zulip in production. 64-bit
+is recommended.
```

Comment:
Huh, yeah I guess so.  Perhaps this should be higher in the page?  I don't know, I wouldn't worry too much about it.